src/main/java/transforms/GeocoderTransform.java

```diff
@@ -68,7 +68,19 @@ public String getAddress( String input, String addressObject ) {
         if ( outercodes.contains( addressObject ) ) {
             return (String) address.get( addressObject );
         } else if ( addressObject.equals("geographypoint") ) {
```

Comment:
Can you move this if to the next line and get rid of the `else if` ? Just easier to read IMO